src/server/typesMap.json

```diff
@@ -19,8 +19,8 @@
             "exclude": [["^", 1, "/.*"]],
             "types": ["office"]
         },
-        "Minified files": {

```

Comment:
Also make this change in editorServices.ts#158

Reply:
done